script/pick_ci.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+case "$(( 0x$(git rev-parse HEAD | cut -c1) % 2 ))" in
```

Comment:
Wouldn't it be a bit clearer to do `echo $(( $RANDOM % 2 ))`? I was initially confused about what `pick_ci` is doing and I don't think there's any predictability in the commit hashes?

Reply:
Ahh scratch that comment, it has to be deterministic across the two builds!

Reply:
Ah yep I can definitely add a comment to make it more obvious what it's doing and why.

And I initially tried without the `cut` because these are just (humongous) numbers 😅